utils/wait/wait.go

```diff
@@ -0,0 +1,81 @@
+package wait
```

Comment:
Please add a copyright header to new files:

```suggestion
// SPDX-License-Identifier: Apache-2.0
// Copyright Authors of Cilium

package wait
```

(I thought there was a `goheader` lint check for this, but it seems like it isn't enabled in this repo)

Reply:
> (I thought there was a `goheader` lint check for this, but it seems like it isn't enabled in this repo)

Addressed in https://github.com/cilium/cilium-cli/pull/1987